documentation/website/src/handbook/integration/serenity-2-migration-guide.md

```diff
@@ -0,0 +1,688 @@
+    * [Background](/handbook/tutorials/serenity-2-migration-guide.html#background)
+    * [How to migrate Promise code](/handbook/tutorials/serenity-2-migration-guide.html#how-to-migrate-promise-code)
+6. [Migrating Activity code](/handbook/tutorials/serenity-2-migration-guide.html#migrating-activity-code)
+  * [Migrating @step annotations](/handbook/tutorials/serenity-2-migration-guide.html#migrating-step-annotations)
+  * [Alert handling](/handbook/tutorials/serenity-2-migration-guide.html#alert-handling)
+    * [Alert related imports](/handbook/tutorials/serenity-2-migration-guide.html#alert-related-imports)
+7. [Migrating Assertions](/handbook/tutorials/serenity-2-migration-guide.html#migrating-assertions)
+  * [chai](/handbook/tutorials/serenity-2-migration-guide.html#chai)
+  * [Migrating See.if to Ensure.that](/handbook/tutorials/serenity-2-migration-guide.html#migrating-see-if-to-ensure-that)
+  * [Is no longer Is :-)](/handbook/tutorials/serenity-2-migration-guide.html#is-no-longer-is-)
+8. [Migrating Questions](/handbook/tutorials/serenity-2-migration-guide.html#migrating-questions)
+9. [Migrating Script Execution Tasks](/handbook/tutorials/serenity-2-migration-guide.html#migrating-script-execution-tasks)
+   * [Serenity/JS 1.x implementation of Hide Task](/handbook/tutorials/serenity-2-migration-guide.html#serenity-js-1-x-implementation-of-hide-task)
+   * [Serenity/JS 2.x implementation of Hide Task](/handbook/tutorials/serenity-2-migration-guide.html#serenity-js-2-0-implementation-of-hide-task)
+
+## Building and viewing documentation locally
+
+Most Serenity/JS classes have been comprehensively documented using [JSDoc](https://devdocs.io/jsdoc/about-getting-started#getting-started) along with usage examples, so this is a very useful documentation source that you can use.
+Now the [2.x website](https://serenity-js.org/) is publicly available, [module documentation](https://serenity-js.org/modules) can be viewed online.
+But just in case you want to build a branch of the documenation locally, you can follow these steps:
+
+* From your terminal window, clone the repository: `git clone https://github.com/jan-molak/serenity-js.git`
+* Move into the project directory: `cd serenity-js`
+* Update dependencies: `npm install`
+* Build the documentation: `make clean verify report site`
+* Move into the build documentation directory: `cd documentation/website`
+* Serve the documentation: `npm start`
+* Open the documentation in your browser at: `http://localhost:3000`
+* For instance, you'll find this guide [here](http://localhost:3000/handbook/tutorials/serenity-2-migration-guide.html), nestled in the [tutorials](http://localhost:3000/handbook/tutorials/index.html)
+
+## Migrating package.json
+### Updating dependencies
+Various libraries are no longer needed. Here's a high-level summary of changes:
+ * Chai is now transitively brought in via `@serenity-js/assertions` 
```

Comment:
Chai is no longer needed as it's been superseded by `@serenity-js/assertions`

Reply:
👍 done